### PR TITLE
Ruby 3 upgrade

### PR DESCRIPTION
**Pull request by grdw**
Bump to ruby 3.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Contributor Covenant Code of Conduct
```
Comment (lorenzograndi4) [inline CODE_OF_CONDUCT.md:1]: what's this code of conduct?

Comment (grdw) [reply CODE_OF_CONDUCT.md:1]: Good question, you get it for free once you do `bundle gem <xyz>`.

Comment (grdw) [reply CODE_OF_CONDUCT.md:1]: We could do without it maybe?

Comment (lorenzograndi4) [reply CODE_OF_CONDUCT.md:1]: ah! ok ,either way 🤷

**Review by lorenzograndi4 — APPROVED** [review]

**Pull request merged**